src/components/Breadcrumbs/Breadcrumbs.scss

```diff
@@ -73,9 +73,12 @@ $stacking-order: (
 }
 
 .Icon {
-  @include recolor-icon(color('ink', 'lighter'));
-  height: $icon-size;
+  @include recolor-icon(
+    var(--p-icon-subdued-on-surface, color('ink', 'lighter'))
+  );
+  z-index: z-index(content, $stacking-order);
```

Comment:
Why is this needed?

Reply:
Because before we were using transparency, and the icon was seen through it. The stacking order was applied to the content, but was missed on the icon and wasn't visible.

Reply:
To keep the existing styling unchanged:

```suggestion
  z-index: var(--p-z-index-content, initial);
```

Reply:
Actually never mind, let’s drop this change for now.